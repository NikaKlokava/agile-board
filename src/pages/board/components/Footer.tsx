import cl from "./styles/footer.module.css";

export const Footer = () => {
  return (
    <div className={cl.footer_wrapper}>
      <p className={cl.footer_text}>Veranika Klokava</p>
      <p className={cl.footer_text}>2023</p>
      <div className={cl.footer_links}>
        <a href="https://instagram.com/nika_klokava?igshid=MmIzYWVlNDQ5Yg==">
          <div className={cl.instagram} />
        </a>
        <a href="https://linkedin.com/in/veranika-klokava-858b5b287">
          <div className={cl.linkedin} />
        </a>
        <a href="https://github.com/NikaKlokava">
          <div className={cl.github} />
        </a>
      </div>
    </div>
  );
};
