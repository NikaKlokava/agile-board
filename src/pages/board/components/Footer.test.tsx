import renderer from "react-test-renderer";
import { Footer } from "./Footer";

describe("Test the Footer component", () => {
  test("The Footer renders correctly", () => {
    const footerSnap = renderer.create(<Footer />).toJSON();
    expect(footerSnap).toMatchSnapshot();
  });
});
