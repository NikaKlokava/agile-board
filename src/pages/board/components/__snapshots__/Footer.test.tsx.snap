// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Footer component The Footer renders correctly 1`] = `
<div
  className="footer_wrapper"
>
  <p
    className="footer_text"
  >
    Veranika Klokava
  </p>
  <p
    className="footer_text"
  >
    2023
  </p>
  <div
    className="footer_links"
  >
    <a
      href="https://instagram.com/nika_klokava?igshid=MmIzYWVlNDQ5Yg=="
    >
      <div
        className="instagram"
      />
    </a>
    <a
      href="https://linkedin.com/in/veranika-klokava-858b5b287"
    >
      <div
        className="linkedin"
      />
    </a>
    <a
      href="https://github.com/NikaKlokava"
    >
      <div
        className="github"
      />
    </a>
  </div>
</div>
`;
